modules/ROOT/partials/integrations/expressjs-quick-start.adoc

```diff
@@ -1,7 +1,7 @@
-This quick start guide walks through creating a https://expressjs.com/en/starter/generator.html[basic Express JS application] containing a {productname} editor based on our link:{baseurl}/demo/basic-example/[Basic example].
+This quick start guide walks through creating a https://expressjs.com/en/starter/generator.html[basic Express JS application] containing a {productname} editor based on our xref:demo/basic-example.adoc[Basic example].
 
 ____
-*NOTE*: {cloudname} users should use the link:{baseurl}/quick-start/[{cloudname} quick start guide].
+*NOTE*: {cloudname} users should use the xref:quick-start.adoc[{cloudname} quick start guide].
 ____
 
 === Prerequisites
```

Comment:
The heading levels in this file are incorrect which means a ToC isn't being generated. This one needs to be something like this:
```suggestion
== Prerequisites
```

Reply:
done